tests/status.spec.coffee

```diff
@@ -100,6 +113,26 @@ describe 'Status', ->
 				key: 'updating'
 				name: 'Updating'
 
+		it 'should return UPDATING if is_online is true and any
+			of the current_services has a download_progress and status is Downloading', ->
```

Comment:
Rather than breaking the line, I think this test needs to have a shorter name. Maybe `should return UPDATING if the device is online with a service downloading`